static/app/views/replays/replayTable/index.tsx

```diff
@@ -215,4 +217,22 @@ const StyledAlert = styled(Alert)`
   margin-bottom: 0;
 `;
 
-export default ReplayTable;
+type AdditionalProps =
+  | {
+      group: Group;
+      showReplayPlayer: true;
+    }
+  | {
+      group?: undefined;
+      showReplayPlayer?: false;
+    };
+
+// Temporary controller to switch between ReplayTable and ReplayTableV2
```

Comment:
i think we'll always need these two distinct tables, because 
- we want the normal replay table (the one shown on replay index) to stay the original table, with no player on top
- the issue details replay table should be the new one you've implemented, which the extra player on top

in that case, i think we only need 1 copy of this `ReplayTable` file, as long as we pass in a prop to `ReplayTable` to determine whether to show the player or not. let me know if this makes sense:

- the V2 table file you've written makes sense to me. it's basically a copy of the original `ReplayTable` but taking in a prop `showReplayPlayer` that determines whether to show the player and other issues-specific things. 
- what if we just had this V2 file be the new `ReplayTable` file? if the prop `showReplayPlayer` is false, then the table will be the same as before -- can use this for replay index, no problem
- if `showReplayPlayer` is true, then we'll show the player, works for issue details
- we can still leverage the feature flag here by conditioning on `showReplayPlayer` AND the feature flag (i.e. both these need to be true to show the player on top, otherwise always return the default table)

either way,  i think we can reuse the same file and don't need 2 copies of the `ReplayTable`.